core/crypto/vrf/p256/p256_test.go

```diff
@@ -45,7 +45,7 @@ func TestH2(t *testing.T) {
 			t.Fatalf("Failed generating random message: %v", err)
 		}
 		x := H2(m)
-		if got := len(x.Bytes()); got < 1 && got > l {
+		if got := len(x.Bytes()); got < 1 || got > l {
```

Comment:
Ok. I see. But then the error message below should be different from:
`t.Errorf("len(h2(%v)) = %v, want %v", m, got, l)`
because this will always say `want 32` 
and not "a number between 1 and N-1"